.devcontainer/cuda11.8-pip/devcontainer.json

```diff
@@ -0,0 +1,49 @@
+{
+  "build": {
+    "context": "${localWorkspaceFolder}/.devcontainer",
+    "dockerfile": "${localWorkspaceFolder}/.devcontainer/Dockerfile",
+    "args": {
+      "CUDA": "11.8",
+      "PYTHON_PACKAGE_MANAGER": "pip",
+      "BASE": "rapidsai/devcontainers:25.04-cpp-cuda11.8-ucx1.18.0-openmpi-ubuntu22.04"
+    }
+  },
+  "runArgs": [
+    "--rm",
+    "--name",
+    "${localEnv:USER:anon}-rapids-${localWorkspaceFolderBasename}-25.04-cuda11.8-pip"
+  ],
+  "hostRequirements": {"gpu": "optional"},
+  "features": {
+    "ghcr.io/rapidsai/devcontainers/features/cuda:25.4": {
+      "version": "11.8",
+      "installcuBLAS": true,
```

Comment:
Do we need these math libraries for UCXX? I don't think we do.

cuDF doesn't have them: https://github.com/rapidsai/cudf/blob/branch-25.04/.devcontainer/cuda11.8-pip/devcontainer.json

Same for CUDA 12.8 pip devcontainers.

Reply:
You're totally right, those are unnecessary here. Removed in https://github.com/rapidsai/ucxx/pull/363/commits/7980f3c0600c2d1544e3f5ccdf683f840e80b024